nullaway/src/main/java/com/uber/nullaway/LibraryModels.java

```diff
@@ -41,6 +41,15 @@
    */
   ImmutableSetMultimap<MethodRef, Integer> failIfNullParameters();
 
+  /**
+   * @return map from the names of methods with @Nullable parameters to the indexes of the arguments
+   *     that are @Nullable.
+   *     <p>This is taken into account for override checks, requiring methods that override the
+   *     methods listed here to take @Nullable parameters on the same indexes. The main use for this
+   *     is to document which API callbacks can be passed null values.
+   */
+  ImmutableSetMultimap<MethodRef, Integer> nullableParameters();
```

Comment:
Maybe `nullableParamsForOverrides()` or `explicitNullableParameters()`?  Would be nice to distinguish more with the default case where we just assume parameters are `@Nullable`

Reply:
👍 Sure! I prefer `explicitNullableParameters()` because we might use them for other things.